app/models/labware_creators/well_filter.rb

```diff
@@ -34,7 +34,7 @@ def filtered_wells
   def filter_requests(requests, well)
     return extract_submission(well) if well.requests_as_source.empty?
 
-    filtered_requests_by_rt = filter_by_request_type(requests)
+    filtered_requests_by_rt = filter_by_request_type(requests.uniq(&:id))
```

Comment:
This is a workaround for duplication of the same request when there are multiple aliquots in the well.

Reply:
Thanks for comments Katy. 
I had a look as well. I think active_requests is a better place to be put in which ensures that duplicates from both requests_as_source and requests_in_progress are removed. Also, this allows deduplication in all methods that use active_requests.